packages/desktop-client/src/components/payees/ManagePayees.tsx

```diff
@@ -290,6 +318,31 @@ export const ManagePayees = ({
           )}
         </View>
       </SelectedProvider>
+
+      <View
+        style={{
+          flexDirection: 'row',
+          justifyContent: 'space-between',
+          margin: '20px 0',
+          flexShrink: 0,
+        }}
+      >
+        <View
+          style={{
+            flexDirection: 'row',
+            alignItems: 'center',
+            gap: '1em',
+          }}
+        >
+          <Button
+            aria-label={t('Change Category Learning')}
+            variant="normal"
+            onPress={onChangeCategoryLearning}
+          >
+            <Trans>Change Category Learning</Trans>
```

Comment:
Something about the "Change" here read strangely to me, but feel free to keep as-is if you prefer it. It's probably a preference thing.

```suggestion
            <Trans>Category learning settings</Trans>
```

If you are keeping it as-is, can we make it lower case?
```suggestion
            <Trans>Change category learning</Trans>
```